service/src/main/java/uk/nhs/adaptors/gp2gp/gpc/GetGpcDocumentTaskDefinition.java

```diff
@@ -1,11 +1,15 @@
 package uk.nhs.adaptors.gp2gp.gpc;
 
+import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.Getter;
 import lombok.experimental.SuperBuilder;
+import lombok.extern.jackson.Jacksonized;
 import uk.nhs.adaptors.gp2gp.common.task.TaskDefinition;
 import uk.nhs.adaptors.gp2gp.common.task.TaskType;
 
+@Data
+@Jacksonized
```

Comment:
Does it work using only the Jacksonzied annotation? Lombok's documentation seems to indicate that it should. https://projectlombok.org/features/experimental/Jacksonized

If Data is required remove the now redundant Getter annotation